src/main/java/uk/gov/companieshouse/pscstatement/delta/mapper/StatementMapper.java

```diff
@@ -39,6 +39,24 @@ default void mapEtag(@MappingTarget Statement target) {
         target.setEtag(GenerateEtagUtil.generateEtag());
     }
 
+    /** Manually map restrictions notice withdrawal reasons. */
+
+    @AfterMapping
+    default void mapRestrictionsNoticeWithdrawalReason(@MappingTarget Statement target, PscStatement source) {
+        if (source.getRestrictionsNoticeReason() != null) {
```

Comment:
I think CHIPS will send null in cases where there's no field but just in case is it worth having an empty check here as well? I think there are common java libraries where isEmpty methods will do null and empty checks.